src/Interfaces/AbstractPolytope.jl

```diff
@@ -96,6 +96,32 @@ function isempty(P::AbstractPolytope)::Bool
     return isempty(vertices_list(P))
 end
 
+"""
+    isuniversal(P::AbstractPolytope{N}, [witness]::Bool=false
+               )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+
+Check whether a polyhedron is universal.
+
+### Input
+
+- `P`       -- polyhedron
+- `witness` -- (optional, default: `false`) compute a witness if activated
+
+### Output
+
+* If `witness` option is deactivated: `false`
+* If `witness` option is activated: `(false, v)` where ``v ∉ P``
+
+### Algorithm
+
+A witness is produced using `isuniversal(H)` where `H` is the first linear
+constraint of `P`.
+"""
+function isuniversal(P::AbstractPolytope{N}, witness::Bool=false
+                    )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+    return witness ? isuniversal(constraints_list(P)[1], true) : false
```

Comment:
The command `constraints_list(P)[1]` will error if `P` has no constraints. As in the case of `HPolyhedron` given above, i would check add the check `if isempty(P.constraints)` here.

Reply:
BTW when is a constraint universal? The `a` has to be zero?

Reply:
A polytope should always have constraints.

Reply:
`HPolytype()` is a valid command..

Reply:
At least, can you add a better error message?